### DISCUSSION TITLE
RSS Feed only for stable Releases

### DISCUSSION THREAD

**Discussion by JokerGermany** [community/community#60433] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Would be great having a feed only for stable releases, like
https://github.com/:owner/:repo/stable-releases.atom

**Comment by jshtuazon2** [comment]
> ### Select Topic Area
> Product Feedback
> 
> ### Body
> Would be great having a feed only for stable releases, like https://github.com/:owner/:repo/stable-releases.atom

**Comment by joooeey** [comment]
With Semantic Versioning being pretty standard now, I think it would be good to give users the option to choose what releases to see on their feed. A few (very few, I suppose) may want to see each development release, others only patch releases, yet others just the minor releases and people might only be interested in the major release.

**Reply by gitcommitshow** [reply]
Agree, `stable-releases` is the only ones I'm interested in. Otherwise the RSS feed is not that useful for.

**Comment by gitcommitshow** [comment]
Agree, it will be better to have this stable release filter. For now, the workaround for me is to use a 3rd party tool https://siftrss.com to filter based on the release title.
